core/src/main/java/org/elasticsearch/common/geo/GeoDistance.java

```diff
@@ -41,7 +41,9 @@
 public enum GeoDistance implements Writeable {
     /**
      * Calculates distance as points on a plane. Faster, but less accurate than {@link #ARC}.
+     * @deprecated use {@link GeoUtils#planeDistance}
      */
+    @Deprecated
```

Comment:
Why are we deprecating and not removing these? We know they don't work in scripts, so what uses could their be other than in our own code?